packages/atomic/src/components/search/search-box-suggestions/atomic-search-box-recent-queries/atomic-search-box-recent-queries.tsx

```diff
@@ -114,13 +114,18 @@ export class AtomicSearchBoxRecentQueries {
     }
 
     const query = this.bindings.searchBoxController.state.value;
+    const querySuggestions =
```

Comment:
A better implementation could be to scope out any other query from the suggestions, whether they are QS or not, at the search box level. We could keep the first occurence of a query & discard of the others.

Reply:
Currently if you add custom suggestions you'd run in the same duplication problem